libs/sdk-ui-kit/src/Dialog/ShareDialog/types.ts

```diff
@@ -0,0 +1,23 @@
+// (C) 2021 GoodData Corporation
+import { IAccessControlAware } from "@gooddata/sdk-backend-spi";
+import { ShareStatus } from "@gooddata/sdk-backend-spi";
+import { IAuditableUsers, ObjRef } from "@gooddata/sdk-model";
+
+/**
+ * @internal
+ */
+export interface IApplyPayload {
```

Comment:
i suggest to rename this

Reply:
Could you give me hint ? Its payload of onApply callback. thx

Reply:
Done